### PR TITLE
Update copyright page

### PR DESCRIPTION
**Pull request by ospsuite**
Explain that manual can be edited by anyone

### CODE REVIEW

**Comment by msevestre** [comment]
I have used an account without any access to our repos to take screenshots etc...

File: copyright.md [hunk 1]
```diff
@@ -17,10 +15,8 @@ The manual is provided “as is” without expressed or implied warranty. You ma
 *	https://computation.llnl.gov/casc/sundials/main.html
```
Comment (Yuri05) [inline copyright.md]: Please also change `(c)2002` to `(c)2015` for the solver (that's the version we're currently using; should upgrade to the latest one at the begin of 2019)

File: copyright.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-Copyright © 2017 Open Systems Pharmacology Suite Community
+Copyright © 2018 Open Systems Pharmacology Suite Community
 
-License Owners of Open Systems Pharmacology Suite are allowed to print and use the PK-Sim® User Manual without fee, provided that the above copyright notice and this paragraph appear in all copies.
-
-The manual is provided “as is” without expressed or implied warranty. You may not modify it in any way.
+The manual is provided “as is” without expressed or implied warranty.
 
 **Trademark information:** All trademarks within this manual belong to their legitimate owners. PK-Sim® and MoBi® use the CVODE solver:
```
Comment (Yuri05) [inline copyright.md]: Please change to "... use CVODE**S** solver..."

File: copyright.md (hunk 1 above)
Comment (Yuri05) [inline copyright.md]: Sundials links are changed and are now:
https://computation.llnl.gov/projects/sundials/license 
https://computation.llnl.gov/projects/sundials

File: copyright.md (hunk 2 above)
Comment (Yuri05) [reply copyright.md]: Also think we should replace `PK-Sim® and MoBi® use the CVODES solver` with `Open Systems Pharmacology Suite uses the CVODES solver`

**Comment by msevestre** [comment]
Going to merge this PR and update your suggestion using my own accounts. It's just easier for advanced git users :)

**Pull request merged**